src/components/Login.js

```diff
@@ -16,7 +18,11 @@ export class Login extends Component {
     handleSubmit = (e) => {
         /* replace the value of v with backend function*/
         const v = this.state.pw === 'password';
```

Comment:
Maybe rename this to verified or isVerified

Reply:
yee sounds good